### PR TITLE
Add support for "Epoch" attributes in RPMs

### PR DESCRIPTION
**Pull request by TvdW**
Fixes https://github.com/bazelbuild/rules_pkg/issues/857

### CODE REVIEW

**Review by cgrindel — APPROVED** [review]
I will wait to merge so that @aiuto can take a look.

File: tests/rpm/BUILD [hunk 1]
```diff
@@ -179,6 +179,7 @@ pkg_rpm(
     architecture = "noarch",
     conflicts = ["not-a-test"],
     description = """pkg_rpm test rpm description""",
+    epoch = "1",
```
Comment (aiuto) [inline tests/rpm/BUILD:182]: Since this is optional, it should not be in every test case.

File: tests/rpm/pkg_rpm_basic_test.py [hunk 2]
```diff
@@ -74,6 +74,7 @@ def test_scriptlet_content(self):
 
     def test_basic_headers(self):
         common_fields = {
+            "EPOCH": b"1",
```
Comment (aiuto) [inline tests/rpm/pkg_rpm_basic_test.py:77]: Can you pass in the check for EPOCH to only the samples which add it?
I know that is not the same pattern here, but what is here was hacked up in a hurry. Having version and release the same for everything is not so bad, but we should have split out noarch and group so they can vary by test case.

**Review by aiuto — CHANGES_REQUESTED** [review]

File: tests/rpm/BUILD (hunk 1 above)
Comment (tvdw-mongodb) [reply tests/rpm/BUILD]: Agree. Updated 😄

Comment (TvdW) [reply tests/rpm/BUILD]: Agree! Updated :smile:

File: tests/rpm/pkg_rpm_basic_test.py (hunk 2 above)
Comment (TvdW) [reply tests/rpm/pkg_rpm_basic_test.py]: Done

**Review by aiuto — APPROVED** [review]
Thanks for the change

**Pull request merged**